crates/ef-testing-proc-macro/src/content_reader.rs

```diff
@@ -0,0 +1,54 @@
+use std::collections::BTreeMap;
+
+use serde_json::Value;
+
+use crate::{dir_reader::PathWrapper, utils::blockchain_tests_to_general_state_tests_path};
+
+pub struct ContentReader;
+
+impl ContentReader {
+    pub fn secret_key(path: PathWrapper) -> Result<Value, eyre::Error> {
```

Comment:
I think this deserves to be documented,

as I understand it, our tests' paths are `BlockchainTests/GeneralStateTests/**/someTest.json` and the secret key is located at `GeneralStateTests/**/someTest.json`

we should provide example and explanation